cmd/BUILD.bazel

```diff
@@ -45,3 +45,12 @@ go_binary(
     embed = [":cmd_lib"],
     visibility = ["//visibility:public"],
 )
+
+alias(
+    name = "prebuilt",
+    actual = select({
+        "@platforms://os:linux": "@bazeldnf-0.5.6-linux-amd64//file",
```

Comment:
is it possible to have an "and" here and select linux and `"@platforms//cpu:x86_64"`?